.github/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
```

Comment:
- [ ] [REQUIRED] Kindly, please make sure to add the `linters.yml` file in the following directory `.github/workflows/linters.yml`